### PR TITLE
[Bug] Fix can't update yarnQueue when update application.

### PR DESCRIPTION
**Pull request by RocMarshal**
## What changes were proposed in this pull request

The logic when update yarn queue of app

## Brief change log

change the logic when update yarn queue of app

## Verifying this change

- *Manually verified the change by testing locally.* 

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): (yes / **no**)

### CODE REVIEW

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/entity/Application.java [hunk 1]
```diff
@@ -133,7 +133,10 @@ public class Application implements Serializable {
   private String module;
 
   private String options;
+
+  @TableField(updateStrategy = FieldStrategy.IGNORED)
```
Comment (RocMarshal) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/entity/Application.java:137]: for `null` value on `hotParams`

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/entity/Application.java [hunk 2]
```diff
@@ -595,6 +598,9 @@ public void doSetHotParams() {
 
   @SneakyThrows
   public void updateHotParams(Application appParam) {
+    if (appParam != this) {
```
Comment (RocMarshal) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/entity/Application.java:601]: to deal the case when set the null or empty queue.

File: streampark-console/streampark-console-webapp/src/views/flink/app/hooks/useCreateAndEditSchema.ts [hunk 3]
```diff
@@ -468,7 +468,7 @@ export const useCreateAndEditSchema = (
         label: t('flink.app.yarnQueue'),
         component: 'Input',
         ifShow: ({ values }) => values.executionMode == ExecModeEnum.YARN_APPLICATION || values.executionMode == ExecModeEnum.YARN_PER_JOB,
-        render: renderYarnQueue
+        render: (renderCallbackParams) => renderYarnQueue(renderCallbackParams)
```
Comment (RocMarshal) [inline streampark-console/streampark-console-webapp/src/views/flink/app/hooks/useCreateAndEditSchema.ts:471]: The bug from Issue-2299

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**